From adf0a917db0c4a809d44601c96b48c4a4804a58f Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 14 Jan 2025 10:53:57 +0800
Subject: [PATCH 18/19] fix(locales): clean up typesetting
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Following localisation guidelines from 《大陆简中自由软件本地化工作指南》
1.5.4 版 (Simplified Chinese - Mainland China FOSS Localization
Guidelines, version 1.5.4).

Link: https://repo.aosc.io/aosc-l10n/zh_CN_l10n_1.5.4.pdf
---
 src/locales/zh.json | 90 ++++++++++++++++++++++-----------------------
 1 file changed, 45 insertions(+), 45 deletions(-)

diff --git a/src/locales/zh.json b/src/locales/zh.json
index 71252903..d11ad1e4 100644
--- a/src/locales/zh.json
+++ b/src/locales/zh.json
@@ -55,12 +55,12 @@
   "Create Profile": "新建配置",
   "Edit Profile": "编辑配置",
   "Edit Proxies": "编辑节点",
-  "Use newlines for multiple uri": "多条URI请使用换行分隔（支持Base64编码）",
+  "Use newlines for multiple uri": "多条 URI 请使用换行分隔（支持 Base64 编码）",
   "Edit Rules": "编辑规则",
   "Rule Type": "规则类型",
   "Rule Content": "规则内容",
   "Proxy Policy": "代理策略",
-  "No Resolve": "跳过DNS解析",
+  "No Resolve": "跳过 DNS 解析",
   "Prepend Rule": "添加前置规则",
   "Append Rule": "添加后置规则",
   "Prepend Group": "添加前置代理组",
@@ -75,26 +75,26 @@
   "DOMAIN-SUFFIX": "匹配域名后缀",
   "DOMAIN-KEYWORD": "匹配域名关键字",
   "DOMAIN-REGEX": "匹配域名正则表达式",
-  "GEOSITE": "匹配Geosite内的域名",
-  "GEOIP": "匹配IP所属国家代码",
-  "SRC-GEOIP": "匹配来源IP所属国家代码",
-  "IP-ASN": "匹配IP所属ASN",
-  "SRC-IP-ASN": "匹配来源IP所属ASN",
-  "IP-CIDR": "匹配IP地址范围",
-  "IP-CIDR6": "匹配IP地址范围",
-  "SRC-IP-CIDR": "匹配来源IP地址范围",
-  "IP-SUFFIX": "匹配IP后缀范围",
-  "SRC-IP-SUFFIX": "匹配来源IP后缀范围",
+  "GEOSITE": "匹配 Geosite 内的域名",
+  "GEOIP": "匹配 IP 所属国家代码",
+  "SRC-GEOIP": "匹配来源 IP 所属国家代码",
+  "IP-ASN": "匹配 IP 所属 ASN",
+  "SRC-IP-ASN": "匹配来源 IP 所属 ASN",
+  "IP-CIDR": "匹配 IP 地址范围",
+  "IP-CIDR6": "匹配 IP 地址范围",
+  "SRC-IP-CIDR": "匹配来源 IP 地址范围",
+  "IP-SUFFIX": "匹配 IP 后缀范围",
+  "SRC-IP-SUFFIX": "匹配来源 IP 后缀范围",
   "SRC-PORT": "匹配请求来源端口范围",
   "DST-PORT": "匹配请求目标端口范围",
   "IN-PORT": "匹配入站端口",
-  "DSCP": "DSCP标记(仅限tproxy udp入站)",
-  "PROCESS-NAME": "匹配进程名称(Android包名)",
+  "DSCP": "DSCP标记（仅限 TPROXY UDP 入站）",
+  "PROCESS-NAME": "匹配进程名称（Android 包名）",
   "PROCESS-PATH": "匹配完整进程路径",
-  "PROCESS-NAME-REGEX": "正则匹配完整进程名称(Android包名)",
+  "PROCESS-NAME-REGEX": "正则匹配完整进程名称（Android 包名）",
   "PROCESS-PATH-REGEX": "正则匹配完整进程路径",
-  "NETWORK": "匹配传输协议(tcp/udp)",
-  "UID": "匹配Linux USER ID",
+  "NETWORK": "匹配传输协议 (TCP/UDP)",
+  "UID": "匹配 Linux USER ID",
   "IN-TYPE": "匹配入站类型",
   "IN-USER": "匹配入站用户名",
   "IN-NAME": "匹配入站名称",
@@ -131,7 +131,7 @@
   "Include All Proxies": "引入所有出站代理",
   "Exclude Filter": "排除节点",
   "Exclude Type": "排除节点类型",
-  "Disable UDP": "禁用UDP",
+  "Disable UDP": "禁用 UDP",
   "Hidden": "隐藏代理组",
   "Group Name Required": "代理组名称不能为空",
   "Group Name Already Exists": "代理组名称已存在",
@@ -147,7 +147,7 @@
   "Choose File": "选择文件",
   "Use System Proxy": "使用系统代理更新",
   "Use Clash Proxy": "使用内核代理更新",
-  "Accept Invalid Certs (Danger)": "允许无效证书 (危险)",
+  "Accept Invalid Certs (Danger)": "允许无效证书（危险）",
   "Refresh": "刷新",
   "Home": "首页",
   "Select": "使用",
@@ -155,7 +155,7 @@
   "Edit File": "编辑文件",
   "Open File": "打开文件",
   "Update": "更新",
-  "Update(Proxy)": "更新(代理)",
+  "Update(Proxy)": "更新（代理）",
   "Confirm deletion": "确认删除",
   "This operation is not reversible": "此操作不可逆",
   "Script Console": "脚本控制台输出",
@@ -193,19 +193,19 @@
   "Test URL": "测试地址",
   "Settings": "设置",
   "System Setting": "系统设置",
-  "Tun Mode": "Tun(虚拟网卡)模式",
+  "Tun Mode": "TUN（虚拟网卡）模式",
   "Reset to Default": "重置为默认值",
-  "Tun Mode Info": "Tun(虚拟网卡)模式接管系统所有流量，启用时无须打开系统代理",
-  "Stack": "Tun 模式堆栈",
+  "Tun Mode Info": "TUN（虚拟网卡）模式接管系统所有流量，启用时无须打开系统代理",
+  "Stack": "TUN 模式堆栈",
   "System and Mixed Can Only be Used in Service Mode": "System 和 Mixed 只能在服务模式下使用",
-  "Device": "Tun 网卡名称",
+  "Device": "TUN 网卡名称",
   "Auto Route": "自动设置全局路由",
   "Strict Route": "严格路由",
   "Auto Detect Interface": "自动选择流量出口接口",
   "DNS Hijack": "DNS 劫持",
   "MTU": "最大传输单元",
   "Service Mode": "服务模式",
-  "Service Mode Info": "启用TUN模式前请安装服务模式，该服务启动的内核进程可获得安装虚拟网卡(TUN模式)的权限",
+  "Service Mode Info": "启用 TUN 模式前请安装服务模式，该服务启动的内核进程可获得安装虚拟网卡（TUN 模式）的权限",
   "Current State": "当前状态",
   "pending": "等待中",
   "installed": "已安装",
@@ -231,9 +231,9 @@
   "Always use Default Bypass": "始终使用默认绕过",
   "Proxy Bypass": "代理绕过设置：",
   "Bypass": "当前绕过：",
-  "Use PAC Mode": "使用PAC模式",
-  "PAC Script Content": "PAC脚本内容",
-  "PAC URL": "PAC地址：",
+  "Use PAC Mode": "使用 PAC 模式",
+  "PAC Script Content": "PAC 脚本内容",
+  "PAC URL": "PAC 地址：",
   "Auto Launch": "开机自启",
   "Silent Start": "静默启动",
   "Silent Start Info": "程序启动时以后台模式运行，不显示程序面板",
@@ -249,14 +249,14 @@
   "Unified Delay": "统一延迟",
   "Unified Delay Info": "开启统一延迟时，会进行两次延迟测试，以消除连接握手等带来的不同类型节点的延迟差异",
   "Log Level": "日志等级",
-  "Log Level Info": "仅对日志目录Service文件夹下的内核日志文件生效",
+  "Log Level Info": "仅对日志目录 Service 文件夹下的内核日志文件生效",
   "Port Config": "端口设置",
   "Random Port": "随机端口",
   "Mixed Port": "混合代理端口",
-  "Socks Port": "Socks代理端口",
-  "Http Port": "Http(s)代理端口",
-  "Redir Port": "Redir透明代理端口",
-  "Tproxy Port": "Tproxy透明代理端口",
+  "Socks Port": "SOCKS 代理端口",
+  "Http Port": "HTTP(S) 代理端口",
+  "Redir Port": "Redir 透明代理端口",
+  "TPROXY Port": "TPROXY 透明代理端口",
   "External": "外部控制",
   "External Controller": "外部控制器监听地址",
   "Core Secret": "API 访问密钥",
@@ -273,7 +273,7 @@
   "Please enter your root password": "请输入您的 root 密码",
   "Grant": "授权",
   "Open UWP tool": "UWP 工具",
-  "Open UWP tool Info": "Windows 8开始限制 UWP 应用（如微软商店）直接访问本地主机的网络服务，使用此工具可绕过该限制",
+  "Open UWP tool Info": "Windows 8 开始限制 UWP 应用（如微软商店）直接访问本地主机的网络服务，使用此工具可绕过该限制",
   "Update GeoData": "更新 GeoData",
   "Verge Setting": "Verge 设置",
   "Language": "语言设置",
@@ -310,9 +310,9 @@
   "Tray Icon": "托盘图标",
   "Common Tray Icon": "常规托盘图标",
   "System Proxy Tray Icon": "系统代理托盘图标",
-  "Tun Tray Icon": "Tun 模式托盘图标",
+  "Tun Tray Icon": "TUN 模式托盘图标",
   "Miscellaneous": "杂项设置",
-  "App Log Level": "App日志等级",
+  "App Log Level": "应用日志等级",
   "Auto Close Connections": "自动关闭连接",
   "Auto Close Connections Info": "当代理组选中节点或代理模式变动时，关闭已建立的连接",
   "Auto Check Update": "自动检查更新",
@@ -322,7 +322,7 @@
   "Auto Columns": "自动列数",
   "Auto Log Clean": "自动清理日志",
   "Never Clean": "不清理",
-  "Retain _n Days": "保留{{n}}天",
+  "Retain _n Days": "保留 {{n}} 天",
   "Default Latency Test": "默认测试链接",
   "Default Latency Test Info": "仅用于 HTTP 客户端请求测试，不会对配置文件产生影响",
   "Default Latency Timeout": "测试超时时间",
@@ -332,11 +332,11 @@
   "clash_mode_global": "全局模式",
   "clash_mode_direct": "直连模式",
   "toggle_system_proxy": "打开/关闭系统代理",
-  "toggle_tun_mode": "打开/关闭 Tun 模式",
+  "toggle_tun_mode": "打开/关闭 TUN 模式",
   "Backup Setting": "备份设置",
-  "Backup Setting Info": "支持WebDAV备份配置文件",
+  "Backup Setting Info": "支持 WebDAV 备份配置文件",
   "Runtime Config": "当前配置",
-  "Open Conf Dir Info": "如果软件运行异常，!备份!并删除此文件夹下的所有文件，重启软件",
+  "Open Conf Dir Info": "如果软件运行异常，*备份*并删除此文件夹下的所有文件，重启软件",
   "Open Conf Dir": "配置目录",
   "Open Core Dir": "内核目录",
   "Open Logs Dir": "日志目录",
@@ -361,11 +361,11 @@
   "Settings Applied": "设置已应用",
   "Service Installed Successfully": "已成功安装服务",
   "Service Uninstalled Successfully": "已成功卸载服务",
-  "Proxy Daemon Duration Cannot be Less than 1 Second": "代理守护间隔时间不得低于1秒",
+  "Proxy Daemon Duration Cannot be Less than 1 Second": "代理守护间隔时间不得低于 1 秒",
   "Invalid Bypass Format": "无效的代理绕过格式",
   "Clash Port Modified": "Clash 端口已修改",
   "Port Conflict": "端口冲突",
-  "Restart Application to Apply Modifications": "重启Verge以应用修改",
+  "Restart Application to Apply Modifications": "重启 Verge 以应用修改",
   "External Controller Address Modified": "外部控制器监听地址已修改",
   "Permissions Granted Successfully for _clash Core": "{{core}} 内核授权成功",
   "Core Version Updated": "内核版本已更新",
@@ -374,7 +374,7 @@
   "GeoData Updated": "已更新 GeoData",
   "Currently on the Latest Version": "当前已是最新版本",
   "Import Subscription Successful": "导入订阅成功",
-  "WebDAV Server URL": "WebDAV服务器地址 http(s)://",
+  "WebDAV Server URL": "WebDAV 服务器地址 http(s)://",
   "Username": "用户名",
   "Password": "密码",
   "Backup": "备份",
@@ -395,7 +395,7 @@
   "Restore Backup": "恢复备份",
   "Backup Time": "备份时间",
   "Confirm to delete this backup file?": "确认删除此备份文件吗？",
-  "Confirm to restore this backup file?": "确认恢复此 份文件吗？",
-  "Restore Success, App will restart in 1s": "恢复成功，应用将在1秒后重启",
+  "Confirm to restore this backup file?": "确认恢复此份文件吗？",
+  "Restore Success, App will restart in 1s": "恢复成功，应用将在 1 秒后重启",
   "Failed to fetch backup files": "获取备份文件失败"
 }
-- 
2.48.0

